### PR TITLE
[MFTF] Adding AdminClearFiltersOnGridActionGroup

### LINKED ISSUE #31632: [Issue] [MFTF] Adding AdminClearFiltersOnGridActionGroup
This issue is automatically created based on existing pull request: magento/magento2#31452: [MFTF] Adding AdminClearFiltersOnGridActionGroup

---------
### Description (*)
Adding AdminClearFiltersOnGridActionGroup

### Related Pull Requests
https://github.com/magento/magento2/pull/31509

### PR DESCRIPTION
**Pull request by AnnaAPak**
### Description (*)
Adding AdminClearFiltersOnGridActionGroup

### Related Pull Requests
https://github.com/magento/magento2/pull/31509

### Resolved issues:
1. [x] resolves magento/magento2#31632: [MFTF] Adding AdminClearFiltersOnGridActionGroup

### CODE REVIEW

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE,Functional Tests EE,Functional Tests B2B

File: app/code/Magento/Backend/Test/Mftf/ActionGroup/AdminClearFiltersOnGridActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminClearFiltersOnGridActionGroup">
```
Comment (eduard13) [inline app/code/Magento/Backend/Test/Mftf/ActionGroup/AdminClearFiltersOnGridActionGroup.xml]: I'd rename it like this. What do you think?
```suggestion
    <actionGroup name="AdminClearGridFiltersActionGroup">
```

File: app/code/Magento/Catalog/Test/Mftf/Test/AdminUpdateTopCategoryUrlWithNoRedirectTest.xml [hunk 2]
```diff
@@ -79,8 +79,8 @@
         <see selector="{{AdminUrlRewriteIndexSection.gridCellByColumnRowNumber('1', 'Request Path')}}" userInput="$$createDefaultCategory.name$$/$$createTwoLevelNestedCategories.name$$/updatedurl.html" stepKey="seeTheRedirectPath"/>
 
         <!-- Verify third level category's old URL path doesn't show redirect path-->
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickOnResetButton1"/>
-        <waitForPageLoad stepKey="waitForPageToLoad3"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickOnResetButton1"/>
```
Comment (eduard13) [inline app/code/Magento/Catalog/Test/Mftf/Test/AdminUpdateTopCategoryUrlWithNoRedirectTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickOnResetButton1"/>
```

File: app/code/Magento/Catalog/Test/Mftf/Test/AdminUpdateTopCategoryUrlWithRedirectTest.xml [hunk 3]
```diff
@@ -77,8 +77,8 @@
         <see selector="{{AdminUrlRewriteIndexSection.gridCellByColumnRowNumber('1', 'Request Path')}}" userInput="$$createDefaultCategory.name$$/$$createTwoLevelNestedCategories.name$$/updateredirecturl.html" stepKey="seeTheRedirectPath"/>
 
         <!-- Verify third level category's Redirect path, Target Path and Redirect type for old URL -->
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickOnResetButton1"/>
-        <waitForPageLoad stepKey="waitForPageToLoad4"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickOnResetButton1"/>
```
Comment (eduard13) [inline app/code/Magento/Catalog/Test/Mftf/Test/AdminUpdateTopCategoryUrlWithRedirectTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickOnResetButton1"/>
```

File: app/code/Magento/Email/Test/Mftf/Test/AdminEmailTemplatePreviewTest.xml [hunk 4]
```diff
@@ -29,7 +29,7 @@
         <after>
             <!--Delete created Template-->
             <actionGroup ref="DeleteEmailTemplateActionGroup" stepKey="deleteTemplate"/>
-            <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clearFilters"/>
+            <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clearFilters"/>
```
Comment (eduard13) [inline app/code/Magento/Email/Test/Mftf/Test/AdminEmailTemplatePreviewTest.xml]:
```suggestion
            <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clearFilters"/>
```

File: app/code/Magento/Email/Test/Mftf/Test/AdminEmailTemplatePreviewWithDirectivesTest.xml [hunk 5]
```diff
@@ -29,7 +29,7 @@
         <after>
             <!--Delete created Template-->
             <actionGroup ref="DeleteEmailTemplateActionGroup" stepKey="deleteTemplate"/>
-            <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clearFilters"/>
+            <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clearFilters"/>
```
Comment (eduard13) [inline app/code/Magento/Email/Test/Mftf/Test/AdminEmailTemplatePreviewWithDirectivesTest.xml]:
```suggestion
            <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clearFilters"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminCreateDefaultsTaxRuleTest.xml [hunk 6]
```diff
@@ -42,7 +42,7 @@
         <see selector="{{AdminTaxRuleGridSection.successMessage}}" userInput="You saved the tax rule." stepKey="assertTaxRuleSuccessMessage" />
 
         <!-- Verify we see created tax rule with defaults(from the above step) on the tax rule grid page -->
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminCreateDefaultsTaxRuleTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRateAllPostCodesTest.xml [hunk 7]
```diff
@@ -43,7 +43,7 @@
 
         <!-- Verify the tax rate grid page shows the tax rate we just created -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex2"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters1"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters1"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRateAllPostCodesTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters1"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRateAllPostCodesTest.xml [hunk 8]
```diff
@@ -54,7 +54,7 @@
 
         <!-- Go to the tax rate edit page for our new tax rate -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex3"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRateAllPostCodesTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRateLargeRateTest.xml [hunk 9]
```diff
@@ -45,7 +45,7 @@
 
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex2"/>
         <!-- Create a tax rate for large postcodes and verify we see expected values on the tax rate grid page -->
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRateLargeRateTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRateSpecificPostcodeTest.xml [hunk 10]
```diff
@@ -45,7 +45,7 @@
 
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex2"/>
         <!-- Verify the tax rate grid page shows the specific postcode we just created -->
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters1"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters1"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRateSpecificPostcodeTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters1"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRateWiderZipCodeRangeTest.xml [hunk 11]
```diff
@@ -45,7 +45,7 @@
 
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex2"/>
         <!-- Create a tax rate for zipCodeRange and verify we see expected values on the tax rate grid page -->
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRateWiderZipCodeRangeTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRateZipCodeRangeTest.xml [hunk 12]
```diff
@@ -46,7 +46,7 @@
 
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex2"/>
         <!-- Create a tax rate for zipCodeRange and verify we see expected values on the tax rate grid page -->
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRateZipCodeRangeTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRuleWithCustomerAndProductTaxClassTest.xml [hunk 13]
```diff
@@ -68,7 +68,7 @@
         <see selector="{{AdminTaxRuleGridSection.successMessage}}" userInput="You saved the tax rule." stepKey="seeAssertTaxRuleSuccessMessage"/>
 
         <!-- Verify we see created tax rule with customer and product class(from the above step) on the tax rule grid page -->
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRuleWithCustomerAndProductTaxClassTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRuleWithNewAndExistingTaxRateAndCustomerAndProductTaxClassTest.xml [hunk 14]
```diff
@@ -63,7 +63,7 @@
         <see selector="{{AdminTaxRuleGridSection.successMessage}}" userInput="You saved the tax rule." stepKey="seeAssertTaxRuleSuccessMessage"/>
 
         <!-- Verify we see created tax rule with new and existing tax rate, customer tax class, product tax class(from the above step) on the tax rule grid page -->
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRuleWithNewAndExistingTaxRateAndCustomerAndProductTaxClassTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRuleWithNewTaxClassesAndTaxRateTest.xml [hunk 15]
```diff
@@ -67,7 +67,7 @@
         <see selector="{{AdminTaxRuleGridSection.successMessage}}" userInput="You saved the tax rule." stepKey="seeAssertTaxRuleSuccessMessage"/>
 
         <!-- Verify we see created tax rule with new tax classes and tax rate(from the above step) on the tax rule grid page -->
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRuleWithNewTaxClassesAndTaxRateTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRuleWithZipRangeTest.xml [hunk 16]
```diff
@@ -65,7 +65,7 @@
         <see selector="{{AdminTaxRuleGridSection.successMessage}}" userInput="You saved the tax rule." stepKey="seeAssertTaxRuleSuccessMessage"/>
 
         <!-- Verify we see created tax rule with zip range(from the above step) on the tax rule grid page -->
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminCreateTaxRuleWithZipRangeTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminDeleteTaxRuleTest.xml [hunk 17]
```diff
@@ -31,7 +31,7 @@
         </after>
 
         <actionGroup ref="AdminTaxRuleGridOpenPageActionGroup" stepKey="goToTaxRuleIndex1"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters1"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters1"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminDeleteTaxRuleTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters1"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminDeleteTaxRuleTest.xml [hunk 18]
```diff
@@ -46,7 +46,7 @@
 
         <!-- Confirm Deleted Tax Rule(from the above step) on the tax rule grid page -->
         <actionGroup ref="AdminTaxRuleGridOpenPageActionGroup" stepKey="goToTaxRuleIndex2"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminDeleteTaxRuleTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminUpdateDefaultTaxRuleTest.xml [hunk 19]
```diff
@@ -39,7 +39,7 @@
             </after>
 
         <actionGroup ref="AdminTaxRuleGridOpenPageActionGroup" stepKey="goToTaxRuleIndex1"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters1"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters1"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminUpdateDefaultTaxRuleTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters1"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminUpdateDefaultTaxRuleTest.xml [hunk 20]
```diff
@@ -64,7 +64,7 @@
         <see selector="{{AdminTaxRuleGridSection.successMessage}}" userInput="You saved the tax rule." stepKey="seeAssertTaxRuleSuccessMessage"/>
 
         <!-- Verify we see updated tax rule with default(from the above step) on the tax rule grid page -->
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminUpdateDefaultTaxRuleTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminUpdateTaxRuleWithCustomClassesTest.xml [hunk 21]
```diff
@@ -40,7 +40,7 @@
         </after>
 
         <actionGroup ref="AdminTaxRuleGridOpenPageActionGroup" stepKey="goToTaxRuleIndex1"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters1"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters1"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminUpdateTaxRuleWithCustomClassesTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters1"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminUpdateTaxRuleWithCustomClassesTest.xml [hunk 22]
```diff
@@ -62,7 +62,7 @@
         <see selector="{{AdminTaxRuleGridSection.successMessage}}" userInput="You saved the tax rule." stepKey="seeAssertTaxRuleSuccessMessage"/>
 
         <!-- Verify we see updated tax rule with default(from the above step) on the tax rule grid page -->
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminUpdateTaxRuleWithCustomClassesTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminUpdateTaxRuleWithFixedZipUtahTest.xml [hunk 23]
```diff
@@ -44,7 +44,7 @@
         </after>
 
         <actionGroup ref="AdminTaxRuleGridOpenPageActionGroup" stepKey="goToTaxRuleIndex1"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters1"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters1"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminUpdateTaxRuleWithFixedZipUtahTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters1"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/AdminUpdateTaxRuleWithFixedZipUtahTest.xml [hunk 24]
```diff
@@ -66,7 +66,7 @@
         <see selector="{{AdminTaxRuleGridSection.successMessage}}" userInput="You saved the tax rule." stepKey="seeAssertTaxRuleSuccessMessage"/>
 
         <!-- Verify we see updated tax rule with fixed zip Utah(from the above step) on the tax rule grid page -->
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/AdminUpdateTaxRuleWithFixedZipUtahTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/DeleteTaxRateEntityTest.xml [hunk 25]
```diff
@@ -36,7 +36,7 @@
 
         <!-- Confirm Deleted TaxIdentifier(from the above step) on the tax rate grid page -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex2"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/DeleteTaxRateEntityTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/Update01TaxRateEntityTest.xml [hunk 26]
```diff
@@ -28,7 +28,7 @@
 
         <!-- Search the tax rate on tax grid page -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex1"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters1"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters1"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/Update01TaxRateEntityTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters1"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/Update01TaxRateEntityTest.xml [hunk 27]
```diff
@@ -45,7 +45,7 @@
 
         <!-- Verify we see updated 0.1 tax rate(from the above step) on the tax rate grid page -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex4"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/Update01TaxRateEntityTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/Update100TaxRateEntityTest.xml [hunk 28]
```diff
@@ -28,7 +28,7 @@
 
         <!-- Search the tax rate on tax grid page -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex1"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters1"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters1"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/Update100TaxRateEntityTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters1"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/Update100TaxRateEntityTest.xml [hunk 29]
```diff
@@ -44,7 +44,7 @@
 
         <!-- Verify we see updated TaxIdentifier(from the above step) on the tax rate grid page -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex4"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/Update100TaxRateEntityTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/Update1299TaxRateEntityTest.xml [hunk 30]
```diff
@@ -28,7 +28,7 @@
 
         <!-- Search the tax identifier on tax grid page -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex1"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters1"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters1"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/Update1299TaxRateEntityTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters1"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/Update1299TaxRateEntityTest.xml [hunk 31]
```diff
@@ -45,7 +45,7 @@
 
         <!-- Verify we see updated tax rate(from the above step) on the tax rate grid page -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex2"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/Update1299TaxRateEntityTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/UpdateAnyRegionTaxRateEntityTest.xml [hunk 32]
```diff
@@ -28,7 +28,7 @@
 
         <!-- Search the tax rate on tax grid page -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex1"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters1"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters1"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/UpdateAnyRegionTaxRateEntityTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters1"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/UpdateAnyRegionTaxRateEntityTest.xml [hunk 33]
```diff
@@ -44,7 +44,7 @@
 
         <!-- Verify we see updated any region tax rate(from the above step) on the tax rate grid page -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex2"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/UpdateAnyRegionTaxRateEntityTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/> 
```

File: app/code/Magento/Tax/Test/Mftf/Test/UpdateDecimalTaxRateEntityTest.xml [hunk 34]
```diff
@@ -28,7 +28,7 @@
 
         <!-- Search the tax rate on tax grid page -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex1"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters1"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters1"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/UpdateDecimalTaxRateEntityTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters1"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/UpdateDecimalTaxRateEntityTest.xml [hunk 35]
```diff
@@ -46,7 +46,7 @@
 
         <!-- Verify we see updated tax rate(from the above step) on the tax rate grid page -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex2"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/UpdateDecimalTaxRateEntityTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/UpdateLargeTaxRateEntityTest.xml [hunk 36]
```diff
@@ -28,7 +28,7 @@
 
         <!-- Search the tax rate on tax grid page -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex1"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters1"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters1"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/UpdateLargeTaxRateEntityTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters1"/>
```

File: app/code/Magento/Tax/Test/Mftf/Test/UpdateLargeTaxRateEntityTest.xml [hunk 37]
```diff
@@ -43,7 +43,7 @@
 
         <!-- Verify we see updated large tax rate(from the above step) on the tax rate grid page -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex4"/>
-        <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
+        <actionGroup ref="AdminClearFiltersOnGridActionGroup" stepKey="clickClearFilters2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/UpdateLargeTaxRateEntityTest.xml]:
```suggestion
        <actionGroup ref="AdminClearGridFiltersActionGroup" stepKey="clickClearFilters2"/>
```

**Review by eduard13 — CHANGES_REQUESTED** [review]
Hi @AnnaAPak, could you please check my minor suggestions below? Please don't forget to rename the ActionGroup file name as well.

Thank you.

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE,Functional Tests EE,Functional Tests B2B

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE

**Comment by eduard13** [comment]
@magento run all tests

**Review by eduard13 — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @eduard13, thank you for the review.
[ENGCOM-8604](https://jira.corp.magento.com/browse/ENGCOM-8604) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
QA not applicable

**Comment by engcom-Charlie** [comment]
@magento create issue

**Pull request merged**